client/view.js

```diff
@@ -73,18 +68,30 @@ export default class View
 
 	#addPlayer(hand)
 	{
+		const $container = document
 		var $player = this.$playerRowTemplate.cloneNode(true);
 		$player.querySelector('.js-id').textContent = hand.playerId;
 		this.#playerRows[hand.playerId] = $player;
 		$player.querySelector('.js-name').textContent = hand.playerName;
-		$player.querySelector('.js-owner').textContent = hand.gameOwner;
+		$player.querySelector('.js-owner').innerHTML = hand.gameOwnerButton;
 		this.$playerTableBody.appendChild($player);
+		$container.querySelector('.js-deal').addEventListener('click', e => {
+			e.preventDefault();
+	
+			if(hand.gameOwner === true)
+			{
+				this.$txtNumJokers = $player.querySelector('.num-jokers')

```

Comment:
its ok to only get a handle on a UI element when you need it (i.e. inside this handler isntead of the ctor), but youre doing both.

Reply:
I'm not following you on this one :)